src/main/java/uk/gov/hmcts/reform/cwrdapi/util/CaseWorkerChildListValidator.java

```diff
@@ -58,6 +60,14 @@ private boolean isValidLocations(CaseWorkerProfile caseWorkerProfile, Constraint
                     .addPropertyNode(LOCATION_FIELD)
                     .addConstraintViolation();
             }
+        } else if (isEmpty(caseWorkerProfile.getLocations())
+                   || (caseWorkerProfile.getLocations().size() == 1
```

Comment:
Sorry for not mentioning this before, I think `caseWorkerProfile.getLocations().size() == 1` is not required here as size > 1 check is being done in the first if condition 
`caseWorkerProfile.getLocations().size() > 1`

Reply:
@SujeethKumarAri  I think this is fine we have 2 locations hence >1  means we must get primary. hence --1 should be fine.

Reply:
@abhijitdiwan - we have two locations and we check > 1 in the first` if`, so it is not mandatory to check if it is ==1 in next `else if `

Reply:
Yes I think @SujeethKumarAri is right, the size == 1 check is redundant at this point, as we know that if it's not empty then the size is definitely 1. Thanks both.

Reply:
Yes correct...We should use empty and last condition with stream to check if primary present or not hence in future we got more locations it won't need change